python/dask_cudf/dask_cudf/backends.py

```diff
@@ -209,8 +210,11 @@ def concat_cudf(
     filter_warning=True,
     sort=None,
     ignore_index=False,
+    ignore_order=False,
```

Comment:
This will break older Dask versions that don't have this parameter... Any ideas?

Reply:
We could use `**kwargs` and `kwargs.get("ignore_order")` internally

Reply:
It shouldn't no? Because we have it as a default param..